hooks/tk-multi-publish2/basic/publish_mari_textures.py

```diff
@@ -248,12 +249,20 @@ def publish(self, settings, item):
         ctx_fields = self.parent.context.as_template_fields(publish_template)
         fields.update(ctx_fields)
 
-        version = 0
+        # For geo name, strip out the non-alphanumeric characters because Mari's
+        # publish template filter does not allow non-alphanumeric characters in
+        # the geo name.
+        pattern = re.compile('[\W_]+', re.UNICODE)
```

Comment:
I'd change these two lines to be:

```python
geo_name = re.sub(r"[\W_]+", "", geo_name)
```

* Do we need `re.UNICODE`?
* Double quotes to align with Tk standards

Reply:
Made changes you requested